components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java

```diff
@@ -45,6 +45,23 @@
     private BpelUIUtil() {
     }
 
+    /**
+     * Truncates a given text to the size specified. Appends "..." to the end.
+     * @param text
+     * @param maxSize
+     * @return Truncated string
+     */
+    public static String truncateString(String text,int maxSize) {
+        long strLength = text.length();
+        if (strLength > maxSize) {
+            if (log.isDebugEnabled()) {
+                log.debug("Limiting variable size of the instance to:"+ maxSize);
+            }
+            return text.substring(0,maxSize) + " ...";
+        }
```

Comment:
Fixed the formatting issues in https://github.com/wso2/carbon-business-process/pull/225